classes/ValidFormBuilder/Textarea.php

```diff
@@ -108,7 +108,14 @@ public function __toHtml($submitted = false, $blnSimpleLayout = false, $blnLabel
         $varValue = $this->__getValue($submitted, $intCount);
 
         $blnError = ($submitted && ! $this->__validator->validate($intCount) && $blnDisplayErrors) ? true : false;
-        if (! $blnSimpleLayout) {
+
```

Comment:
Why did you only implement this for `TextArea`? I would expect an `Element` implementation that works for all field types.